lib/main.js

```diff
@@ -37,7 +38,7 @@ fse.ensureDirSync(options.tmpdir)
 
 const utils = require('./utils')
 const Resinio = utils.requireComponent('resinio', 'sdk')
-const resinos = utils.requireComponent('resinos', 'default')
+const Resinos = utils.requireComponent('os', 'resinos')
```

Comment:
Lets use a simple `require()` here, as we're not dealing with any combinations

Reply:
We are not dealing with combinations now, but probably in the future we will.